src/common.build.pre.props

```diff
@@ -63,8 +72,8 @@
       <!--<WarningLevel>EnableAllWarnings</WarningLevel>-->
       <TreatWarningAsError>true</TreatWarningAsError>
       <!-- disable warning on nameless structs (4201) -->
-      <DisableSpecificWarnings>4201;4312;4467</DisableSpecificWarnings>
-      <PreprocessorDefinitions>EXTERNAL_BUILD;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <DisableSpecificWarnings>4201;4312;4467;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <PreprocessorDefinitions>_WINDOWS;EXTERNAL_BUILD;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
What does `_WINDOWS` do?

Reply:
No idea, but we had it elsewhere so I merged it into here.